http/src/main/resources/reference.conf

```diff
@@ -202,7 +202,7 @@ azure {
         minor-version-auto-upgrade = true,
         file-uris = ["https://raw.githubusercontent.com/DataBiosphere/leonardo/1b7690cde14132d6478d5c014341a01b557c6132/http/src/main/resources/init-resources/azure_vm_init_script.sh"]
       }
-      listener-image = "terradevacrpublic.azurecr.io/terra-azure-relay-listeners:8fe77a0"
+      listener-image = "terradevacrpublic.azurecr.io/terra-azure-relay-listeners:decc8b0"
```

Comment:
This new version includes this bugfix: https://github.com/DataBiosphere/terra-azure-relay-listeners/pulls?q=is%3Apr+is%3Aclosed